dicelog.py

```diff
@@ -71,7 +71,7 @@ def dicelog(bot, trigger):
         rollStr = trigger.group(2).strip()
     campaign = campaign.strip()
     rollStr = rollStr.strip()
-    # prepare string for mathing
+    # prepare string for matching
```

Comment:
This one's intended. https://www.urbandictionary.com/define.php?term=mathing

Reply:
KK, will fix